pages/Guides/supernode-program/docs.en.md

```diff
@@ -6,92 +6,122 @@ taxonomy:
 
-  3 million XEM to be able to participate in the program, and 30 XEM to be able to pay the fees for activating delegated harvesting and sending the enrollment message. The message fee depends on the size of the message text, so 30 XEM might be a little bit more than you really need.
+**Bandwidth**
+The bandwidth test checks a node is running on hardware with a sufficiently capable network connection. The controller selects the partner node with the lowest ping time as measured by the ping test and passes it to the node under test via the ``nr/task/bandwidth`` API. The controller also sends a random hash seed. The node then sends the random hash seed to the partner, which hashes it 30000 times and returns a list of all resulting hashes concatenated to the seed. The sending node measures the time needed to complete this operation. Finally, the sending node independently performs the same hash calculation and returns the results to the controller. This test passes when the hashes calculated by the node and the partner node are equal and the measured operation time indicates a transfer speed of at least 5 Mbit/s.
 
-- **Step 3**: Activate delegated harvesting on your main account.
+**Responsiveness**
+The responsiveness test checks how quickly a node can respond to a chain height request. The controller initiates 10 chain height requests via the ``/chain/height`` API. It then calculates the total amount of time to fulfill all of the requests. This test passes when at least 9 requests are fulfilled and the total time to fulfill all requests was no greater than one second. If only the time threshold is exceeded, the controller will retry this test up to 4 times.
 
-  A tutorial can be found [here](http://nem.ghost.io/how-to-use-delegated-harvesting/).
+## Steps to Enroll and Participate in the Supernode Program
+
+- **Step 1**: Download and Launch Latest NEM NanoWallet From [the GitHub Releases Page](https://github.com/NemProject/NanoWallet/releases).
+
+- **Step 2**: Make an Account and Have at Least 10,010 XEM.
+
+  10,000 XEM to be able to participate in the Supernode Program, and ~10 XEM to pay the fees for activating delegated harvesting and sending the enrollment message. (You will need to re-enroll in the Supernode Program every month - 10 XEM should be enough to cover transaction fees for at least a year.)
+
+- **Step 3**: Activate Delegated Harvesting on Your Main Account.
+
+  The tutorial for NEM NanoWallet delegated harvesting can be found [here](https://nemproject.github.io/nem-docs/pages/Guides/nanowallet/delegated-harvesting/docs.en.html#activation).
+
+  Six (6) hours must pass before you can start harvesting, but enrollment in the Supernode Program can be initiated once you have the delegated private key (available immediately). (Please make sure to get the private key from the **delegated** harvesting account, not your main account and not either of the public keys.)
```

Comment:
Can be left as it, but I feel the Delegated key is overstated in this section. I would consider removing "(Please make sure to get the private key from the **delegated** harvesting account, not your main account and not either of the public keys.)" as this is covered at least twice directly below. Just my take though - this can be left as is; I understand that it is an important point.

Not sure what's rendering this content, but another approach would be an 'admonition' box like the note and warning boxes in https://docs.symbol.dev/guides/network/connecting-to-mongodb.html - could help to more clearly call out & emphasize the point.

```
.. note:: _Note Contents_
.. warning: _Warning Contents_
```

.. warning::

Reply:
- removed `"(Please make sure to get the private key from the delegated harvesting account, not your main account and not either of the public keys.)`
- changed already existing sentence to warning:
![image](https://user-images.githubusercontent.com/17684281/183021866-60661450-04dc-4bcb-a533-c0fd33f66bb4.png)